runners/spark/src/main/java/org/apache/beam/runners/spark/SparkRunner.java

```diff
@@ -143,40 +146,27 @@ private SparkRunner(SparkPipelineOptions options) {
   public EvaluationResult run(Pipeline pipeline) {
     try {
       LOG.info("Executing pipeline using the SparkRunner.");
-      JavaSparkContext jsc;
-      if (mOptions.getUsesProvidedSparkContext()) {
```

Comment:
Moved [here](https://github.com/apache/incubator-beam/pull/909/files#diff-cdc52a99e44ad1b6d1950ecab0928ecdR49).
It had to take into account reuse for testing.